plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/reviews-title/edit.tsx

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { useEntityProp } from '@wordpress/core-data';
+import { PanelBody, ToggleControl } from '@wordpress/components';
+import { useState, useEffect } from '@wordpress/element';
+import { resolveSelect } from '@wordpress/data';
+import { reviewsStore } from '@woocommerce/data';
+import {
+	// @ts-expect-error AlignmentControl is not exported from @wordpress/block-editor
+	AlignmentControl,
+	BlockControls,
+	useBlockProps,
+	InspectorControls,
+	// @ts-expect-error HeadingLevelDropdown is not exported from @wordpress/block-editor
+	HeadingLevelDropdown,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import type { ProductReviewsTitleEditProps } from './types';
+
+function getProductReviewsTitle(
+	showReviewsCount: boolean,
+	reviewsCount: number,
+	showProductTitle: boolean,
+	productTitle: string
+) {
+	if ( showReviewsCount && showProductTitle ) {
+		return reviewsCount === 1
+			? sprintf(
+					/* translators: %s: Post title. */
```

Comment:
Nit: Can we update this in all places? 

```suggestion
					/* translators: %s: Product title. */
```

Reply:
Done, thanks: https://github.com/woocommerce/woocommerce/pull/56332/commits/f643843317f856cfad884a280fbb0fb194d3d1c7